extra-security/clipass/autobuild/build

```diff
@@ -0,0 +1 @@
+npm i -g --user root --prefix="$PKGDIR/usr" clipass
```

Comment:
Please add an `abinfo` here to describe what is being done. This has been specified in the [AOSC OS Package Styling Manual](https://wiki.aosc.io/en/dev-sys-package-styling-manual).

Reply:
Ack.

Reply:
Added. Does it look good?